node-operators/validators/onboarding/opt-in-to-tanssi.md

```diff
@@ -0,0 +1,274 @@
+
+If you have already [registered as an operator](/node-operators/validators/onboarding/register-in-symbiotic/){target=\_blank}, and before starting to secure Tanssi-powered networks, you must register your intention by opting into a Tanssi-enabled Symbiotic vault and the Tanssi network. Both steps require interacting with smart contracts, one from Symbiotic's core protocol and the other from Tanssi's integration with Symbiotic.
+
+--8<-- 'text/node-operators/validators/onboarding/run-a-validator/prerequisites.md'
+
+## Opt-In to Tanssi-Enabled Vaults {: #opt-in-tanssi-vaults }
+
+Before enabling your operator to be an active Tanssi network validator, you must opt in to at least one of the Tanssi-enabled vaults. The following sections describe several ways to opt into the vaults.
+
+### Opt-In Using the Symbiotic CLI {: #opt-in-tanssi-vaults-with-cli }
+
+If you have correctly installed the [Symbiotic CLI](#setting-up-the-cli) and you want to sign the transaction using a ledger device, then run the following command, replacing the `VAULT_ADDRESS` with the specific one you want to join and `OPERATOR_ADDRESS` with your account:
+
+=== "Mainnet"
+
+    ```bash
+    python3 symb.py opt-in-vault VAULT_ADDRESS --ledger --ledger-account OPERATOR_ADDRESS
+    ```
+
+=== "Testnet (Sepolia)"
+
+    ```bash
+    python3 symb.py opt-in-vault {{ networks.symbiotic.contracts.sepolia.vault }} --ledger --ledger-account OPERATOR_ADDRESS --chain sepolia
+    ```
+
+If you want to sign the transaction directly using the operator's account private key, then run the following command, replacing the `PRIVATE_KEY` parameter:
+
+=== "Mainnet"
+
+    ```bash
+    python3 symb.py opt-in-vault VAULT_ADDRESS --private-key PRIVATE_KEY
+    ```
+
+=== "Testnet (Sepolia)"
+
+    ```bash
+    python3 symb.py opt-in-vault {{ networks.symbiotic.contracts.sepolia.vault }} --private-key PRIVATE_KEY --chain sepolia
+    ```
+
+!!! warning
+    Note that this method requires you to expose your private key; therefore, it is not recommended.
+
+### Opt-In Using Etherscan {: #opt-in-tanssi-vaults-with-etherscan }
+
+You can interact with Symbiotic's smart contracts using Etherscan, and sign the transaction using a browser wallet ([Metamask](/builders/toolkit/ethereum-api/wallets/metamask/#install-the-metamask-extension){target=\_blank}, for example). 
+
+To open the contrat's page, open the link:
+
+=== "Mainnet"
+
+    [https://etherscan.io/address/{{ networks.symbiotic.contracts.mainnet.opt_in_vault }}#writeContract](https://etherscan.io/address/{{ networks.symbiotic.contracts.mainnet.opt_in_vault }}#writeContract){target=\_blank}
+
+=== "Testnet (Sepolia)"
+
+    [https://sepolia.etherscan.io/address/{{ networks.symbiotic.contracts.sepolia.opt_in_vault }}#writeContract](https://sepolia.etherscan.io/address/{{ networks.symbiotic.contracts.sepolia.opt_in_vault }}#writeContract){target=\_blank}
+
+Click on `Connect to Web3`, and select your preferred wallet (e.g. Metamask):
```

Comment:
```suggestion
Click on **Connect to Web3**, and select your preferred wallet (e.g. MetaMask):
```

UI elements should be bold.

Please update remainder of Metamask instances to MetaMask and update all UI elements to be bold